src/components/Header/NavDropdown.tsx

```diff
@@ -50,8 +32,11 @@ const DropdownItem: React.VFC<DropdownItemType> = ({
   to,
 }) => (
   <MenuItem
-    as={to && LinkWrapper}
-    data-to={to}
+    as={to && Link}
+    // LG typing should permit props associated with the `as`
+    // component, but right now it doesn't. ¯\_(ツ)_/¯
+    // @ts-expect-error
+    to={to}
```

Comment:
Maybe this is covered by [LG-2609](https://jira.mongodb.org/browse/LG-2609)?

Reply:
Yeah, and there's also [LG-2161](https://jira.mongodb.org/browse/LG-2161) that aims to replace box... I think things should be looking up in this area!